similarity-data-impl/src/main/java/org/phenotips/data/similarity/internal/AbstractPatientGenotypeSimilarityView.java

```diff
@@ -93,7 +93,7 @@ public boolean hasGenotypeData()
             Set<String> matchGenes = this.matchGenotype.getCandidateGenes();
             Set<String> refGenes = this.refGenotype.getCandidateGenes();
             sharedGenes.addAll(matchGenes);
-            sharedGenes.retainAll(refGenes);
+            sharedGenes.addAll(refGenes);
```

Comment:
This looks wrong. `sharedGenes` means genes that both patients have in common, not the union of their genes.